NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/model/Payroll.java

```diff
@@ -1,11 +1,8 @@
 package com.nullterminators.project.model;
 
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.Id;
-import jakarta.persistence.Table;
-import jakarta.persistence.GenerationType;
+import jakarta.persistence.*;
```

Comment:
Why did you replace with .*? Checkstyle does not support .* imports

Reply:
Corrected it!